Mathlib/Algebra/Equiv/TransferInstance.lean

```diff
@@ -5,6 +5,7 @@ Authors: Johannes Hölzl
 -/
 import Mathlib.Algebra.Algebra.Equiv
 import Mathlib.Algebra.Field.Basic
+import Mathlib.Algebra.Group.TransferInstance
 import Mathlib.Logic.Equiv.Defs
 import Mathlib.Logic.Small.Defs
 import Mathlib.Algebra.Ring.Hom.InjSurj
```

Comment:
Rather than repeating the docstring of `Group.TransferInstance`, let's replace it with something short like "this continues the pattern set in `Mathlib/Algebra/Group/TransferInstance.lean`, so that we can describe it in detail in just one place.